### PR TITLE
chore: fix stress test log

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

**Review by sandrask — APPROVED** [review]

File: test/bdd/stress_steps.go [hunk 1]
```diff
@@ -381,11 +384,16 @@ func (r *createDIDReq) Invoke() (interface{}, error) {
 		return nil, err
 	}
 
-	logger.Infof("created did successfully %s", intermID)
+	createLogCount++
```
Comment (bstasyszyn) [inline test/bdd/stress_steps.go:387]: I think you need to use atomic.AddInt64, otherwise you'll get a race.

File: test/bdd/stress_steps.go [hunk 2]
```diff
@@ -381,11 +384,16 @@ func (r *createDIDReq) Invoke() (interface{}, error) {
...
+
+	if createLogCount%100 == 0 {
```
Comment (bstasyszyn) [inline test/bdd/stress_steps.go:389]: Need to use atomic.LoadInt64.

File: test/bdd/stress_steps.go [hunk 3]
```diff
@@ -436,7 +443,11 @@ func (r *resolveDIDReq) Invoke() (interface{}, error) {
 
 	canonicalID := docResolution.DocumentMetadata.CanonicalID
 
-	logger.Infof("resolved created did successfully %s", canonicalID)
+	resolveCreateLogCount++
```
Comment (bstasyszyn) [inline test/bdd/stress_steps.go:446]: Same comment as above.

File: test/bdd/stress_steps.go (hunk 2 above)
Comment (bstasyszyn) [reply test/bdd/stress_steps.go:389]: Actually, you don't need to load - just use the new value returned from Add.

Comment (fqutishat) [reply test/bdd/stress_steps.go]: done

File: test/bdd/stress_steps.go (hunk 3 above)
Comment (fqutishat) [reply test/bdd/stress_steps.go]: done

File: test/bdd/stress_steps.go (hunk 1 above)
Comment (fqutishat) [reply test/bdd/stress_steps.go]: done

**Review by bstasyszyn — APPROVED** [review]

**Pull request merged**